File: examples/3_usando_modulos/ex018_angulo_seno_cosseno_tangente/ex018.py
# Faça um programa que leia um angulo qualquer e mostre na tela o valor do seno, cosseno e tangente desse ângulo.

from math import cos, sin, tan

angle = int(input('Digite o valor do ângulo a ser calculado seno, cosseno e tangente: '))
print(f'O seno de {angle}° é = {sin(angle)}')
print(f'O cosseno de {angle}° é = {cos(angle)}')
print(f'A tangente de {angle}° é = {tan(angle)}')

File: examples/3_usando_modulos/ex018_angulo_seno_cosseno_tangente/ex018.js
function calculate() {

    let angle = document.querySelector('#input1').value;
    let angleValue = Number(angle);
    let radians = angleValue * Math.PI/180;
    let divAnswer = document.querySelector('.answer');
    
    if (angle == '' || angleValue <= 0 || angleValue > 360 || isNaN(angleValue)) {

        divAnswer.innerHTML = `<div class="answer error">Informe um ângulo válido!</div>`;

    } else {

        divAnswer.innerHTML = '';
        divAnswer.innerHTML += `<div class="answer">O seno de ${angleValue}° é igual a ${Math.sin(radians).toFixed(3)}</div>`;
        divAnswer.innerHTML += `<div class="answer">O cosseno de ${angleValue}° é igual a ${Math.cos(radians).toFixed(3)}</div>`;
        divAnswer.innerHTML += `<div class="answer">A tangente de ${angleValue}° é igual a ${Math.tan(radians).toFixed(3)}</div>`;

    }
}